pkg/venafi/cloud/certificatePolicies.go

```diff
@@ -94,29 +94,43 @@ func (ct certificateTemplate) toPolicy() (p endpoint.Policy) {
 		return s
 	}
 	addStartEndToArray := func(ss []string) []string {
+		// if the array is nil, return nil
+		if ss == nil {
+			return nil
+		}
+
 		a := make([]string, len(ss))
 		for i, s := range ss {
 			a[i] = addStartEnd(s)
 		}
 		return a
 	}
-	if len(ct.SubjectCValues) == 0 {
-		ct.SubjectCValues = []string{".*"}
```

Comment:
I do not know why this custom logic was here.
On first sight, this does not seem to be correct behavior.

Reply:
This line was introduced in https://github.com/Venafi/vcert/commit/4429c01ca2ef6a09ab52c29dc4115c0c658bb931.